README.md

```diff
@@ -1,2 +1,45 @@
 # iexec-commons-poco
 PoCo Java wrappers for the iExec platform
+
+
+## Steps to generate Java Wrappers for Smart Contracts
+
+First you need to install last version of Web3j CLI, the simplest way to install the Web3j CLI is via the following script:
+
+```
+curl -L get.web3j.io | sh && source ~/.web3j/source.sh
+```
+Verify the installation was successful 
+```
+web3j -v 
+Version: 1.4.2
+```
+
+After, you need to clone `Poco-dev` repository and compile
+```
+git clone https://github.com/iExecBlockchainComputing/PoCo-dev/
+cd PoCo-dev
+git checkout x.y.z
+npm i
+./node_modules/.bin/truffle compile
+```
+
+```
+web3j generate truffle --truffle-json ~/iexecdev/PoCo-dev/build/contracts/App.json -o ~/iexecdev/iexec-commons-poco/src/main/java/ -p com.iexec.commons.poco.contract.generated
```

Comment:
Use script for main "easy" block?
```
./generateContractWrappers \
  App \
  AppRegistry \
  [...]
```

Reply:
it's a good idea, I can make a small bash script